vortex-vector/src/vector_mut.rs

```diff
@@ -66,10 +66,9 @@ impl VectorMut {
             DType::Decimal(decimal_dtype, _) => {
                 DecimalVectorMut::with_capacity(decimal_dtype, capacity).into()
             }
-            DType::Utf8(_)
-            | DType::Binary(_)
-            | DType::List(..)
-            | DType::Extension(_) => vortex_panic!("Unsupported dtype for VectorMut"),
+            DType::Utf8(_) | DType::Binary(_) | DType::List(..) | DType::Extension(_) => {
```

Comment:
Don't we have a CI check for formatting, how did this land?

Reply:
Only way things like that land is a semantic merge conflict